en/docs/includes/deploy/assign-custom-hostname.md

```diff
@@ -18,13 +18,13 @@
 
 3.  Click on the API to edit its configurations.
 
-     [![Edit the API](/../../../../assets/img/includes/deploy/select-api.png)](/../../../../assets/img/includes/deploy/select-api.png)
+     [![Edit the API](../../assets/img/includes/deploy/select-api.png)](../../assets/img/includes/deploy/select-api.png)

```

Comment:
This change is wrong pls revert it.

Reply:
the image is not rendering in https://apim.docs.wso2.com/en/latest/deploy-and-publish/deploy-on-gateway/deploy-api/exposing-apis-via-custom-hostnames/

Reply:
reverted with https://github.com/wso2/docs-apim/pull/3842/commits/013b5cef34ccd5cbfe5edccf52d59d8a67dbb6dd